src/libs/Log.ts

```diff
@@ -82,4 +83,15 @@ const Log = new Logger({
 });
 timeout = setTimeout(() => Log.info('Flushing logs older than 10 minutes', true, {}, true), 10 * 60 * 1000);
 
+AppLogs.configureAppGroupName('group.com.expensify.new');
+AppLogs.registerHandler({
+    filter: '[NotificationService]',
+    handler: ({filter, logs}) => {
+        logs.forEach((log) => {
+            // sanitize filter to avoid infinite loop
+            Log.info(log.message.replace(filter, '[PushNotification]'));
```

Comment:
We also have a few `.error` level logs in the NotificationService like [this one](https://github.com/margelo/expensify-app-fork/blob/461733465d40f436c6f50fca3457458b8a053377/ios/NotificationServiceExtension/NotificationService.swift#L221). Can we check and use `Log.hmmm` for those instead?

Reply:
@arosiclair done 🙌